diff -purN a/share/efi.tmpl b/share/efi.tmpl
--- a/share/efi.tmpl	2018-12-07 13:18:52.677753417 -0500
+++ b/share/efi.tmpl	2018-12-07 13:23:01.835309042 -0500
@@ -8,9 +8,21 @@ APPLE_EFI_DISKNAME=inroot+"/usr/share/pi
 
 mkdir ${EFIBOOTDIR}
 mkdir ${EFIBOOTDIR}/fonts/
+%if exists("boot/efi/EFI/*/shim.efi"):
 install boot/efi/EFI/*/shim.efi ${EFIBOOTDIR}/BOOT${efiarch}.EFI
+%else:
+install boot/efi/EFI/*/shim${efiarch|lower}.efi ${EFIBOOTDIR}/BOOT${efiarch}.EFI
+%endif
+%if exists("boot/efi/EFI/*/MokManager.efi"):
 install boot/efi/EFI/*/MokManager.efi ${EFIBOOTDIR}/
+%else:
+install boot/efi/EFI/*/mm${efiarch|lower}.efi ${EFIBOOTDIR}/
+%endif
+%if exists("boot/efi/EFI/*/gcd" + efiarch.lower() + ".efi"):
 install boot/efi/EFI/*/gcd${efiarch|lower}.efi ${EFIBOOTDIR}/grub${efiarch|lower}.efi
+%else:
+install boot/efi/EFI/*/grub${efiarch|lower}.efi ${EFIBOOTDIR}/grub${efiarch|lower}.efi
+%endif
 install boot/efi/EFI/*/fonts/unicode.pf2 ${EFIBOOTDIR}/fonts/
 %if exists("usr/share/shim/lockdown.efi"):
     install usr/share/shim/lockdown.efi ${EFIBOOTDIR}/
@@ -47,9 +59,9 @@ ${make_efiboot("images/efiboot.img")}
     replace @INITRDPATH@ /${kdir}/initrd.img ${eficonf}
     replace @ISOLABEL@ '${isolabel}' ${eficonf}
     %if disk:
-        replace @ROOT@ inst.stage2=hd:LABEL=ANACONDA ${eficonf}
+        replace @ROOT@ 'ks=hd:LABEL=ANACONDA:/${product.name}.ks inst.stage2=hd:LABEL=ANACONDA' ${eficonf}
     %else:
-        replace @ROOT@ 'inst.stage2=hd:LABEL=${isolabel|udev}' ${eficonf}
+        replace @ROOT@ 'ks=hd:LABEL=${isolabel|udev}:/${product.name}.ks inst.stage2=hd:LABEL=${isolabel|udev}' ${eficonf}
     %endif
     %if efiarch == 'IA32':
         copy ${eficonf} ${EFIBOOTDIR}/BOOT.conf
