diff -u -r a/src/pylorax/installer.py b/src/pylorax/installer.py
--- a/src/pylorax/installer.py	2019-06-27 15:09:27.000000000 -0400
+++ b/src/pylorax/installer.py	2020-04-16 23:35:47.622167590 -0400
@@ -392,10 +392,13 @@
             log.info(line)
 
         # Make sure the new filesystem is correctly labeled
+        fc_tmpl = "/etc/selinux/%s/contexts/files/file_contexts"
         setfiles_args = ["-e", "/proc", "-e", "/sys", "-e", "/dev",
-                         "/etc/selinux/targeted/contexts/files/file_contexts", "/"]
+                         "", "/"]
 
         if "--dirinstall" in args:
+            poltype = execWithCapture("python3", ["-c", "import selinux; print(selinux.selinux_getpolicytype())"], root=dirinstall_path)
+            setfiles_args[-2] = fc_tmpl % poltype.strip()
             # setfiles may not be available, warn instead of fail
             try:
                 execWithRedirect("setfiles", setfiles_args, root=dirinstall_path)
@@ -404,6 +407,8 @@
         else:
             with PartitionMount(disk_img) as img_mount:
                 if img_mount and img_mount.mount_dir:
+                    poltype = execWithCapture("python3", ["-c", "import selinux; print(selinux.selinux_getpolicytype())"], root=img_mount.mount_dir)
+                    setfiles_args[-2] = fc_tmpl % poltype.strip()
                     try:
                         execWithRedirect("setfiles", setfiles_args, root=img_mount.mount_dir)
                     except (subprocess.CalledProcessError, OSError) as e:
