diff -purN a/imgcreate/kickstart.py b/imgcreate/kickstart.py
--- a/imgcreate/kickstart.py	2016-02-01 14:59:36.172089443 -0500
+++ b/imgcreate/kickstart.py	2016-02-01 14:58:41.103088175 -0500
@@ -477,7 +477,9 @@ def get_timeout(ks, default = None):
         return default
     return int(ks.handler.bootloader.timeout)
 
-def get_kernel_args(ks, default = "ro liveimg quiet"):
+# SRS: drop quiet since we really want noisy boot info
+#def get_kernel_args(ks, default = "ro liveimg quiet"):
+def get_kernel_args(ks, default = "ro liveimg live_ram"):
     if not hasattr(ks.handler.bootloader, "appendLine"):
         return default
     if ks.handler.bootloader.appendLine is None:
diff -purN a/imgcreate/live.py b/imgcreate/live.py
--- a/imgcreate/live.py	2016-02-01 14:59:39.064084816 -0500
+++ b/imgcreate/live.py	2016-02-01 14:58:41.104088226 -0500
@@ -103,9 +103,13 @@ class LiveImageCreatorBase(LoopImageCrea
         """
         r = kickstart.get_kernel_args(self.ks)
         if os.path.exists(self._instroot + "/usr/bin/rhgb"):
-            r += " rhgb"
+            # SRS: disable RHGB because it is unreliable across a variety of vid cards
+            #r += " rhgb"
+            r += ""
         if os.path.exists(self._instroot + "/usr/bin/plymouth"):
-            r += " rhgb"
+            # SRS: disable RHGB because it is unreliable across a variety of vid cards
+            #r += " rhgb"
+            r += ""
         return r
         
     def _get_mkisofs_options(self, isodir):
@@ -614,9 +618,10 @@ menu hiddenrow 5
                                                timeout = self._timeout * 10)
 
         cfg += self.__get_image_stanzas(isodir)
-        cfg += self.__get_memtest_stanza(isodir)
-        cfg += self.__get_local_stanza(isodir)
-        cfg += self._get_isolinux_stanzas(isodir)
+        # SRS: remove unused menu opts
+        #cfg += self.__get_memtest_stanza(isodir)
+        #cfg += self.__get_local_stanza(isodir)
+        #cfg += self._get_isolinux_stanzas(isodir)
 
         cfgf = open(isodir + "/isolinux/isolinux.cfg", "w")
         cfgf.write(cfg)
