diff -purN a/imgcreate/kickstart.py b/imgcreate/kickstart.py
--- a/imgcreate/kickstart.py	2016-02-01 13:40:26.216092704 -0500
+++ b/imgcreate/kickstart.py	2016-02-01 13:40:30.020095465 -0500
@@ -426,7 +426,7 @@ class SelinuxConfig(KickstartConfig):
 
     def apply(self, ksselinux):
         if os.path.exists(self.path("/usr/sbin/lokkit")):
-            args = ["/usr/sbin/lokkit", "-f", "--quiet", "--nostart"]
+            args = ["/usr/sbin/lokkit", "--quiet", "--nostart"]
 
             if ksselinux.selinux == ksconstants.SELINUX_ENFORCING:
                 args.append("--selinux=enforcing")
