diff -purN a/src/pypungi/__init__.py b/src/pypungi/__init__.py
--- a/src/pypungi/__init__.py	2017-04-14 11:34:35.162539929 -0400
+++ b/src/pypungi/__init__.py	2017-04-14 11:56:04.517707341 -0400
@@ -923,7 +923,6 @@ class Pungi(pypungi.PungiBase):
                 pprint.pformat(list(sorted(failed)))))
             self.logger.info("Couldn't find %i of %i srpms." % (
                 len(failed), len(self.src_by_bin)))
-            raise RuntimeError("Could not find all srpms.")
 
     def add_srpms(self, po_list=None):
         """Cycle through the list of package objects and
@@ -934,10 +933,14 @@ class Pungi(pypungi.PungiBase):
         po_list = po_list or self.po_list
         for po in sorted(po_list):
             srpm_po = self.sourcerpm_srpmpo_map[po.sourcerpm]
-            if srpm_po in self.completed_add_srpms:
+            try:
+                srpm_po = self.sourcerpm_srpmpo_map[po.sourcerpm]
+            except KeyError:
+                self.logger.error("Cannot get source RPM '%s' for %s" % (po.sourcerpm, po.nvra))
+                srpm_po = None
+
+            if srpm_po is None:
                 continue
-            msg = "Added source package %s.%s (repo: %s)" % (srpm_po.name, srpm_po.arch, srpm_po.repoid)
-            self.add_source(srpm_po, msg)
 
             # flags
             if po in self.input_packages:
@@ -949,6 +952,12 @@ class Pungi(pypungi.PungiBase):
             if po in self.multilib_packages:
                 self.multilib_packages.add(srpm_po)
 
+            if srpm_po in self.completed_add_srpms:
+                continue
+ 
+            msg = "Added source package %s.%s (repo: %s)" % (srpm_po.name, srpm_po.arch, srpm_po.repoid)
+            self.add_source(srpm_po, msg)
+
             self.completed_add_srpms.add(srpm_po)
             srpms.add(srpm_po)
         return srpms
