diff -purN pungi-2.0.22/src/bin/pungi.py pungi-2.0.22.new/src/bin/pungi.py
--- pungi-2.0.22/src/bin/pungi.py	2010-08-24 21:21:51.000000000 -0400
+++ pungi-2.0.22.new/src/bin/pungi.py	2015-03-31 13:44:05.882027626 -0400
@@ -170,6 +170,9 @@ if __name__ == '__main__':
         parser.add_option("--bugurl", dest="bugurl", type="string",
           action="callback", callback=set_config, callback_args=(config, ),
           help='the url for your bug system (defaults to http://bugzilla.redhat.com)')
+        parser.add_option("--updates", dest="updates", type="string",
+          action="callback", callback=set_config, callback_args=(config, ),
+          help='include anaconda updates in rolled iso (see http://fedoraproject.org/wiki/Anaconda/Updates)')
         parser.add_option("--selfhosting", action="store_true", dest="selfhosting",
           help='build a self-hosting tree by following build dependencies (optional)')
         parser.add_option("--fulltree", action="store_true", dest="fulltree",
diff -purN pungi-2.0.22/src/pypungi/__init__.py pungi-2.0.22.new/src/pypungi/__init__.py
--- pungi-2.0.22/src/pypungi/__init__.py	2010-08-24 21:20:38.000000000 -0400
+++ pungi-2.0.22.new/src/pypungi/__init__.py	2015-03-31 13:49:38.529143267 -0400
@@ -778,6 +778,10 @@ class Pungi(pypungi.PungiBase):
             buildinstall.append('--bugurl')
             buildinstall.append(self.config.get('pungi', 'bugurl'))
 
+        if self.config.has_option('pungi', 'updates'):
+            buildinstall.append('--updates')
+            buildinstall.append(self.config.get('pungi', 'updates'))
+
         buildinstall.append('--output')
         buildinstall.append(self.topdir)
 
