diff -purN pungi-2.0.22/src/pypungi/__init__.py pungi-2.0.22.new/src/pypungi/__init__.py
--- pungi-2.0.22/src/pypungi/__init__.py	2015-03-31 14:34:34.069050701 -0400
+++ pungi-2.0.22.new/src/pypungi/__init__.py	2015-03-31 14:36:24.961085152 -0400
@@ -1054,7 +1054,7 @@ cost=500
 
         # setup the base command
         mkisofs = ['/usr/bin/mkisofs']
-        mkisofs.extend(['-v', '-U', '-J', '-R', '-T', '-m', 'repoview', '-m', 'boot.iso']) # common mkisofs flags
+        mkisofs.extend(['-f', '-v', '-U', '-J', '-R', '-T', '-m', 'repoview', '-m', 'boot.iso']) # common mkisofs flags
 
         x86bootargs = ['-b', 'isolinux/isolinux.bin', '-c', 'isolinux/boot.cat', 
             '-no-emul-boot', '-boot-load-size', '4', '-boot-info-table']
